fetch/main.js

```diff
@@ -0,0 +1,13 @@
+async function getData(url = '', data = []) {
```

Comment:
What's the purpose of the `data` parameter? It must be leftover from something you did earlier.

Reply:
Corrected